examples/generated-sources/dub.json

```diff
@@ -2,6 +2,6 @@
 	"name": "generated-sources",
 	"description": "Example of using pre generate commands to generate source code.",
 	"preGenerateCommands": [
-		"echo int fun() { return 42; } > source/test.d"
+		"echo 'int fun() { return 42; }' > source/test.d"
```

Comment:
Why are the ticks added? I just tested it and the file `test.d` is created as
```d
'int fun() { return 42; }' 
```

Reply:
I can confirm that the ticks cause problems, at least on Windows.